### PR TITLE
Fix docs broken links

### PR DESCRIPTION
**Pull request by luisorbaiceta**
Due to some reports about broken links, I'm opening this PR to fix all missing or old links.

### Broken Links list:

- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Guides/Fluent-Schema.md?plain=1#L62) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Guides/Serverless.md?plain=1#L67) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Guides/Serverless.md?plain=1#L96) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Guides/Serverless.md?plain=1#L114) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Guides/Style-Guide.md?plain=1#L73) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Decorators.md?plain=1#L75) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Decorators.md?plain=1#L103) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Decorators.md?plain=1#L104) wrong link
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Errors.md?plain=1#L99) #bodyLimit should be #bodylimit
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Plugins.md?plain=1#L15) wrong #tag
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Plugins.md?plain=1#L33) should be #route-prefixing-option
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Reply.md?plain=1#L70) There is not a #Request tag (remove)
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Reply.md?plain=1#L513) # should be #onsend
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Routes.md?plain=1#L85) # should be #seterrorhandler
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Routes.md?plain=1#L110) #version does not exist
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Routes.md?plain=1#L440) # should be #serializers-object
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Server.md?plain=1#L1078) # no longer exists
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Server.md?plain=1#L1294) make relative (wrong absolute path)
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Server.md?plain=1#L1295) make relative (wrong absolute path)
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Server.md?plain=1#L1405) make relative (wrong absolute path)
- [x] [docs](https://github.com/fastify/fastify/blob/627f7bde9a5d483f68b765668ea4fe964ab41cbf/docs/Reference/Validation-and-Serialization.md?plain=1#L716) make relative (wrong absolute path)

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Guides/Serverless.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ if (require.main === module) {
 
 When executed in your lambda function we do not need to listen to a specific
 port, so we just export the wrapper function `init` in this case. The
-[`lambda.js`](https://www.fastify.io/docs/latest/Serverless/#lambda-js) file
+[`lambda.js`](#lambdajs) file
```
Comment (jsumners) [inline docs/Guides/Serverless.md:67]: I'm pretty sure there was a reason for the full URL link, but I can't recall it.

**Review by jsumners — APPROVED** [review]
Looks good to me.

Everything in this PR is exactly why we need to migrate to a system, like Hugo, that support proper references. Yes, it would mean the docs are not easily readable in the git/GitHub repository. But it would solve all of these errors in a much more manageable way.

**Comment by luisorbaiceta** [comment]
> Looks good to me.
> 
> Everything in this PR is exactly why we need to migrate to a system, like Hugo, that support proper references. Yes, it would mean the docs are not easily readable in the git/GitHub repository. But it would solve all of these errors in a much more manageable way.

In my opinion a huge step forward would be to restrict internal linking within the docs to relative links

File: docs/Guides/Serverless.md (hunk 1 above)
Comment (luisorbaiceta) [reply docs/Guides/Serverless.md:67]: I thought so to, but couldn't find any

**Pull request merged**